DEV-GUIDE.md

```diff
@@ -69,6 +69,15 @@ To run some of Istio's examples and tests, you need to set up Docker server.
 Please follow [these instructions](https://docs.docker.com/engine/installation/)
 for how to do this for your platform.
 
+Ensure your UID is in the docker group to access the docker daemon as a non-root user:
+
+```shell
+sudo adduser username docker
```

Comment:
you could just put $USER or $(id -un)

Reply:
ya - I can change this to sudo -E adduser $USER docker ($USER iirc is overridden within a sudo environment)

Reply:
`sudo adduser $USER docker` works because $USER is expanded by the current shell, not by sudo